File: redcap_v15.7.6/API/examples/ruby/export_logging.rb
#!/usr/bin/env ruby

require 'curl'
require './settings.rb'
include Settings

fields = {
  :token => Settings::API_TOKEN,
  :content => 'log',
  :format => 'json',
  :logtype => '',
  :user => '',
  :record => '',
  :beginTime => '10/06/2020 17:37',
  :endTime => ''
}

ch = Curl::Easy.http_post(
  Settings::API_URL,
  fields.collect{|k, v| Curl::PostField.content(k.to_s, v)}
)
puts ch.body_str

File: redcap_v15.7.6/API/examples/r/export_logging.R
#!/usr/bin/env Rscript

source('config.R')
library(RCurl)

result <- postForm(
    api_url,
    token=api_token,
    content='log',
    format='json',
    logtype='',
    user='',
    record='',
    beginTime='10/06/2020 17:37',
    endTime=''
)
print(result)
